File: tests/testdata/codegen/nested-invokes-pp/dotnet/nested-invokes.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using Std = Pulumi.Std;

return await Deployment.RunAsync(() => 
{
    var example = Std.Replace.Invoke(new()
    {
        Text = Std.Upper.Invoke(new()
        {
            Input = "hello_world",
        }).Result,
        Search = "_",
        Replace = "-",
    });

});

File: tests/testdata/codegen/nested-invokes-pp/go/nested-invokes.go
package main

import (
	"github.com/pulumi/pulumi-std/sdk/go/std"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		_, err := std.Replace(ctx, &std.ReplaceArgs{
			Text: std.Upper(ctx, &std.UpperArgs{
				Input: "hello_world",
			}, nil).Result,
			Search:  "_",
			Replace: "-",
		}, nil)
		if err != nil {
			return err
		}
		return nil
	})
}
